docs/_layouts/guides.html

```diff
@@ -32,6 +32,31 @@ <h2 class="guides-post-title"><a href="{{ post.url | relative_url }}"
                                 <a href="{{ post.url | relative_url }}" class="read-more-link">Read more</a>
                             </div>
                         </div>
+                    </div> -->
+
+                    <div class="media">
```

Comment:
I like the layout of the cards, and think the objectives look great now. 

One personal bit of feedback, is on mobile I'd prefer the text to be left/centre aligned to take up more of the card.

<img width="418" alt="Screenshot 2020-06-03 at 14 34 27" src="https://user-images.githubusercontent.com/31372187/83643100-8b18c800-a5a7-11ea-8715-0fbb16c345b8.png">